docs/self-managed/setup/install.md

```diff
@@ -52,6 +52,23 @@ Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md
 
 When installing the [camunda-platform](https://artifacthub.io/packages/helm/camunda/camunda-platform) Helm chart, all components shown on the architectural diagram above are installed.
 
+## Licensing
+
+Camunda 8 components are now able to consume licensing information with the following Helm configuration:
+
+```yaml
+## @extra global.license
+license:
+  ## @param global.license.key if set, it will be exposed as "CAMUNDA_LICENSE_KEY" in all components, consumable as ENV_VAR.
+  key:
+  ## @param global.license.existingSecret you can provide an existing secret name for Camunda license secret.
+  existingSecret:
+  ## @param global.license.existingSecretKey you can provide the key within the existing secret object for Camunda license key.
+  existingSecretKey:
+```
+
+This means Camunda 8 components may request valid licensing within the logs, and you may notice a "no license" label in the navigation bar. However, this has no impact on startup.
```

Comment:
```suggestion
Camunda 8 components without a valid license may display **Non-Production License** in the navigation bar and issue warnings in the logs. These warnings have no impact on startup.
```

I know this change is in /next, but do you think it's worth calling out somehow that licenses don't/won't exist for enterprise customers yet? My concern is someone reading these docs or seeing the warning and worrying they are an enterprise customer and have no license, but I'm not sure how valid that fear is.

Reply:
Agreed on these changes 👍